### DISCUSSION TITLE
Settings proxy envs for pdm install

### DISCUSSION THREAD

**Discussion by Nomiez** [pdm-project/pdm#2732] [Q&A]
Hey, I have a small probem. I want to use PDM in my company behind a proxy. Currently I have to export the proxy environemnt variables `http_proxy` and `https_proxy` manually each time I want to use pdm. Therefore I am searching for a way that is equivalent to ryes proxy config in the .toml file for pdm:

```
[proxy]
# the proxy to use for HTTP (overridden by the http_proxy environment variable)
http = "http://127.0.0.1:4000"
# the proxy to use for HTTPS (overridden by the https_proxy environment variable)
https = "http://127.0.0.1:4000"
```
Is there some option to do it also like this in PDM? It would also be sufficient if they could be set before executing a pdm command like pdm install. Thanks for your help :)

**Comment by pawamoy** [comment]
Can't you set these environment variables as permanent on your system?

**Reply by Nomiez** [reply]
Sadly no, there are some internal applications that I cannot access via the proxy :/

**Reply by pawamoy** [reply]
I see, thanks. Well, feel free to open a feature request :)
